### PR TITLE
Pointed to Jenkins plugin enhancement documentation

### PR DESCRIPTION
**Pull request by bmcelvee**
https://trello.com/c/MYUfyDqm/144-document-additional-jenkins-plugin-enhancement-requests-from-github

### CODE REVIEW

**Comment by bmcelvee** [comment]
@gabemontero Would you mind verifying the links to the 'OpenShift V3 Plugin for Jenkins' README.md are correct? 

Preview: http://file.rdu.redhat.com/~bmcelvee/07042017/jenkins_plugin_enhancements/using_images/other_images/jenkins.html#openshift-origin-pipeline-plug-in

**Comment by gabemontero** [comment]
hey @bmcelvee 

The "SCM-style polling" link should be to https://github.com/openshift/jenkins-plugin#jenkins-source-code-management-scm

Otherwise looks good.

**Comment by bmcelvee** [comment]
Thanks, @gabemontero!

**Comment by bmcelvee** [comment]
@mburke5678 quick peer review, please?

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -400,18 +400,25 @@ For more details on the sample job included in this image, see this link:https:/
 
 The Jenkins image's list of pre-installed plug-ins includes a plug-in which
 assists in the creating of CI/CD workflows that run against an {product-title}
-server.  A series of build steps, post-build actions, as well as SCM-style
-polling are provided which equate to administrative and operational actions on
+server.  A series of
+link:https://github.com/openshift/jenkins-plugin#jenkins-build-steps[build
+steps],
+link:https://github.com/openshift/jenkins-plugin#jenkins-post-build-actions[post-build
+actions], as well as
```
Comment (mburke5678) [inline using_images/other_images/jenkins.adoc]: s/as well as/and

File: using_images/other_images/jenkins.adoc [hunk 2]
```diff
@@ -400,18 +400,25 @@ For more details on the sample job included in this image, see this link:https:/
...
+link:https://github.com/openshift/jenkins-plugin#jenkins-source-code-management-scm[SCM-style
+polling] are provided which equate to administrative and operational actions on
```
Comment (mburke5678) [inline using_images/other_images/jenkins.adoc]: s/provided/provided,

**Comment by mburke5678** [comment]
A couple of nits. 
I am not crazy about the "which equate to" on line 409, but not sure what to do with it.

**Comment by bmcelvee** [comment]
Thanks @mburke5678! I'm not crazy about the "which equate to" bit either, but I might need to revisit this section later.

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../using_images/other_images/jenkins.adoc#using-images-other-images-jenkins[Jenkins]
|Added links to the OpenShift V3 Plugin for Jenkins README.md documentation in the xref:../using_images/other_images/jenkins.adoc#pipeline-plug-in[{product-title} Pipeline Plug-in] section.
%

**Pull request merged**